roles/workstation/tasks/apt-repos1.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
The name of this file should be more interesting. This remeinds me of Final draft final final (1)(2021)

Lets add something to this. Either add the version of node to the other one or isolate all the nodes into a single apt file and call it apt-node-repos or the like.

Reply:
I updated the file names to reflect more clearly what they are for
5ab91ba67441c9caf6e8668ff523c0df811316cd